src/views/virtualmachinesinstance/details/tabs/details/components/Details/HadwareDevices/HardwareDevices.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+import { V1Devices } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+
+type HardwareDevices = {
+  devices: V1Devices;
+};
+
+const HardwareDevices: React.FC<HardwareDevices> = ({ devices }) => {
+  const { t } = useKubevirtTranslation();
+  const gpuDevices = devices?.gpus?.length || 0;
+  const hostDevices = devices?.hostDevices?.length || 0;
+
+  return (
+    <div>
+      <div className="text-muted">{t('{{gpuDevices}} GPU Devices', { gpuDevices })}</div>
```

Comment:
suggestion: replace all `div` elements with the `className="text-muted"` with the `MutedTextDiv` in the src/utils/components directory